src/App.tsx

```diff
@@ -16,35 +26,121 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([...goodsFromServer]);
+  const [sortState, setSortState] = useState<{
+    reversed: boolean;
+    criterion: SortType;
```

Comment:
```suggestion
    isReversed: boolean;
    sortType: SortType;
```
Boolean variables should start from `is` or `has`